core/controllers/editor.py

```diff
@@ -716,7 +701,7 @@ def post(self, exploration_id):
 
         file_system_class = fs_services.get_exploration_file_system_class()
         fs = fs_domain.AbstractFileSystem(file_system_class(
-            'exploration/%s' % exploration_id))
+            'exploration', exploration_id))
```

Comment:
We ought to use a named constant like fs_domain.ENTITY_TYPE_EXPLORATION for this; ditto elsewhere.

Reply:
Done